Logbook.org

```diff
@@ -0,0 +1,293 @@
+#+title: Logbook
+#+author: @CFGrote:github.com
+#+date: 20241212
+
+* Dec. 10 2024
+   :PROPERTIES:
+   :CUSTOM_ID: dec.-10-2024
+   :ID:       af2db540-6116-4fb3-9f02-33368cb8456a
+   :END:
+** Run 1
+:PROPERTIES:
+:ID:       ea102cdd-2c30-4892-91db-67a39d9e906d
+:END:
+*** Testing all queries from remote via vpn
+:PROPERTIES:
+:ID:       1749151c-81d9-4d5d-9e3c-d2dd8b21d95d
+:END:
+- Servers: ontop, fuseki
+- number of query repeats: 30
+- client ip: 128.176.254.12
+- client hostname: micropop046
+- client software: apache-jena-5.2.0/bin/rsparql
+- rsparql version reported as 4.2.0
+- connection: vpn to uni-muenster
+- note: all queries were run once about 30min before. Noting reduced
+  query time on the 00-construct and other queries. Caching?
```

Comment:
We can add there was no available API for controlling caching within servers so the best we could was hot cache?

Reply:
done in 32df07f